Tests/DependencyInjection/AbstractDoctrineExtensionTest.php

```diff
@@ -294,6 +304,9 @@ public function testLoadSingleConnection()
         $this->assertDICDefinitionMethodCallOnce($configDef, 'setDefaultRepositoryClassName', ['Acme\Doctrine\Repository']);
     }
 
+    /**
+     * @group legacy
```

Comment:
we need to split this test in 2. We need a non-legacy test covering the loading of multiple connections (using non-deprecated settings for the cache).